test/com/google/javascript/jscomp/CollapsePropertiesTest.java

```diff
@@ -1617,92 +1917,91 @@ public void testLocalAliasOfEnumWithInstanceofCheck() {
         " var Fruit = null;\n" +
         " if (f == Enums$Fruit$APPLE) alert('apple');\n" +
         " if (f == Enums$Fruit$BANANA) alert('banana');\n" +
-        "}",
-        null);
```

Comment:
It looks like this `null` was already removed a while ago, so the patch is not applying cleanly. I think if you merge or rebase that will fix it?